core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsInput.java

```diff
@@ -39,57 +42,87 @@
 /**
  * GtfsInput provides a common interface for reading GTFS data, either from a ZIP archive or from a directory.
```

Comment:
Javadoc needs to be updated for abstract class vs. interface

Reply:
An abstract class is still providing an interface, so I would like to keep the existing phasing..